pipelineutilities/pipelineutilities/standard_json_helpers.py

```diff
@@ -221,3 +222,25 @@ def _add_sequence(standard_json: dict) -> dict:
     if 'sequence' not in standard_json:
         standard_json['sequence'] = 0
     return standard_json
+
+
+def _insert_pdf_images(standard_json: dict):
+    tif_items = []
+    for item in standard_json.get('items', []):
+        if item.get('level') == 'file' and item.get('id', '').lower().endswith('.pdf'):
+            if 'sequence' not in item:
+                item['sequence'] = 1
+            item_clone = dict(item)
+            item_clone['id'] = item_clone.get('id').replace('.pdf', '.tif')
```

Comment:
In addition to 'id', could you also please replace '.pdf' with '.tif' for nodes in this list: 
['id', 'filePath', 'description', 'title']

I think this will cover all cases of where and how this may be used.